commerce_coordinator/apps/ecommerce/serializers.py

```diff
@@ -6,9 +6,11 @@ class OrderCreatedSignalInputSerializer(serializers.Serializer):  # pylint: disa
     """
     Serializer for order_created_signal input validation.
     """
-    coupon_code = serializers.CharField(allow_null=True)
-    edx_lms_user_id = serializers.IntegerField(allow_null=False)
-    email = serializers.EmailField(allow_null=False)
     product_sku = serializers.ListField(
         child=serializers.CharField(), allow_empty=False
     )
+    edx_lms_user_id = serializers.IntegerField(allow_null=False)
+    email = serializers.EmailField(allow_null=False)
+    first_name = serializers.CharField(allow_null=True, allow_blank=True)
+    last_name = serializers.CharField(allow_null=True, allow_blank=True)
```

Comment:
`first_name` and `last_name` are optional fields in the `User` table, Which is why I keep them nullable. 
Ideally, the Titan system should not raise errors if these fields are blank. We need to confirm with Titans.

Reply:
These are required field for Titans, We are passing dummy values for now.